opennms-base-assembly/src/main/filtered/etc/events/opennms.pollerd.events.xml

```diff
@@ -267,8 +267,8 @@
    <event>
       <uei>uei.opennms.org/nodes/nodeCategoryMembershipChanged</uei>
       <event-label>OpenNMS-defined node event: nodeCategoryMembershipChanged</event-label>
-      <descr>&lt;p>Node category membership has changed for node
-            (%parm[nodelabel]%)&lt;/p></descr>
+      <descr>&lt;p>Node (%parm[nodelabel]%) has changed its
+          membership from (%parm[categoriesDeleted]%) to (%parm[categoriesAdded]%).&lt;/p></descr>
```

Comment:
This message makes no sense, because categoriesDeleted and categoriesAdded are sets of deleted/added categories and not the sets of categories the node was member of before and after addition/deletion of categories. So, if your node X has memberships A,B,C,D and D will be deleted and E added it will result in: "Node X has changed its membership from (D) to (E)." which is obviously wrong, because the node memberships are now A,B,C,E.

Reply:
Is there a parm with the list of current categories? Maybe it would be useful to say "node categories X deleted, new active node categories is <current-list>"

Reply:
No there is none. And it is not trivial to add...